### DISCUSSION TITLE
Não acho devs React em Curitiba

### DISCUSSION THREAD

**Discussion by matheus-rosin** [frontendbr/forum#1463] [Discussão]
Sou de Curitiba, e preciso de devs React para trampar alocados na mesma empresa que eu. Anunciei (bem como a galera do RH) em vários lugares, e vieram apenas 2 de Curitiba.

Tem algum lugar massa de anunciar, onde eu possa encontrar mais? Tá difícil a coisa rs

**Comment by mdandrade** [comment]
Conhece o techjobs?

___________________________________
Atenciosamente,

Márcio Daniel Andrade

________________________________

**Comment by rafaelangical** [comment]
se rolar remoto, tô disponível, só avisar.

**Comment by matheus-rosin** [comment]
@mdandrade Não conheço não, vou dar uma olhada
@rafaelangical Estamos preferindo alocados, mas se não conseguir encontrar ninguém, vai ter que ser remoto mesmo... Daí vemos o que rola ;)

**Comment by felipefialho** [comment]
Tentou no https://github.com/frontendbr/vagas e https://github.com/frontendbr/me-contrata?

**Comment by matheus-rosin** [comment]
@felipefialho sim... no primeiro, vieram vários, mas não de Curitiba; no segundo, achei um que tem React na descrição, mas pelos projetos, precisava dar uma madurada ainda :/

**Comment by felipemfp** [comment]
> @felipefialho sim... no primeiro, vieram vários, mas não de Curitiba; no segundo, achei um que tem React na descrição, mas pelos projetos, precisava dar uma madurada ainda :/

E dentro da empresa não dá para dá essa madurada não?

**Comment by matheus-rosin** [comment]
Então cara... Por mim, seria tranquilo. Mas conhecendo a empresa, e vendo a enxurrada de demanda e a situação em geral, certeza que não daria certo :(

**Comment by luzimar** [comment]
Tenho disponibilidade

**Comment by matheus-rosin** [comment]
@luzimar Curitiba?

**Comment by luzimar** [comment]
não sou de Curitiba, mas dependendo da oportunidade, posso me mudar.

**Comment by carloshenriqueribeiro** [comment]
vulpi
revelo
programathor

**Comment by carloshenriqueribeiro** [comment]
Veja também se a remuneração que oferecem está compatível com o mercado.

**Comment by matheus-rosin** [comment]
@luzimar manda e-mail lá, matheus@personaliza.com.br
@carloshenriqueribeiro esses aí já usamos; chegamos a nem colocar remuneração, para evitar isso

**Comment by vsalbuq** [comment]
@matheus-rosin , Estou fazendo o bootcamp da Rocketseat e eles costumam divulgar vagas internamente para as tecnologias da OmniStack (Node, React e React Native).

Já pensou em divulgar lá? Tem um canal na comunidade do Discord apenas para vagas:

https://discordapp.com/channels/327861810768117763/384757111873077248

**Comment by evertonthepaula** [comment]
https://www.meetup.com/pt-BR/ReactJS-CWB/

**Comment by matheus-rosin** [comment]
@vsalbuq um outro parceiro aqui já abriu, e nada
@evertonthepaula já abrimos também rs

**Comment by guibfo** [comment]
> Então cara... Por mim, seria tranquilo. Mas conhecendo a empresa, e vendo a enxurrada de demanda e a situação em geral, certeza que não daria certo :(

Ao meu ver se você não está conseguindo ninguém ninguém mesmo, ou você vai ter que tornar sua proposta mais atraente, pra conseguir levar alguém com o nível de conhecimento que vocês querem, ou levar alguém pra aprender mesmo

Esses dias rolou [essa vaga aqui](https://github.com/frontendbr/vagas/issues/2103) que você pode ver como foi bem recebida aí pela comunidade, levanta essa bola aí pra galera de levar alguém pra aprender, as vezes você se surpreende

**Comment by matheus-rosin** [comment]
@guibfo esse é o b.o., conflito de interesses da empresa. Em relação a pegar para aprender, a questão é tempo. Hoje temos dois devs aqui, e um vai sair. Vai ficar um. Precisamos de uns 4, 5 devs novos para pegar a vaga desse que vai sair, de um outro que saiu, e de novos projetos. Isso porque é o que toparam contratar, porque na real precisava de uns 7, 8 rss
Sobre o salário, o big boss ainda precisa ver que precisa melhorar rs mas agora simplesmente retiramos os limites salariais (digo, não limitamos mais para ver o que apareceria), e nem assim

**Comment by felipefialho** [comment]
@matheus-rosin e já pensou em ter outra abordagem?

React é JavaScript, ao invés de buscar experts em React (uma lib), de repente, procurar bons profissionais JavaScript e dar tempo para essas pessoas aprenderem a lib.

**Comment by matheus-rosin** [comment]
@felipefialho seria o ideal! No entanto... rsss
Entenda que quando digo estas coisas, digo como que fora de mim. Por mim, a coisa era certa; o problema é o big boss aceitar, entende? Acontece que vários projetos foram aceitos pela empresa, e com tendo apenas dois devs! (e um ainda vai sair em alguns dias...)
Ou seja, se as coisas tivessem sido programadas corretamente, não haveria problema em qualquer das abordagens citadas. Mas já que não rolou, agora precisamos de gente para sair fazendo.

**Comment by matheus-rosin** [comment]
Não sei ao certo, mas parece que o problema maior está em Curitiba. Tem bastante dev para Angular, WordPress e PHP. De resto, está difícil achar (precisamos de um backer para Node.js). Vem gente de vários lugares do Brasil, mas de Curitiba, quase ninguém

**Comment by alexgwolff** [comment]
Se rolar um remoto @matheus-rosin  tenho interesse como PJ ou cooperado.
Segue meu linkedin : https://www.linkedin.com/in/alexgwolff/

**Comment by matheus-rosin** [comment]
@alexgwolff massa, cara! Vou manter aqui. Por ora, não está rolando remoto; mas em breve creio que conseguiremos, daí falo contigo ;)

**Comment by kivervinicius** [comment]
puta oportunidade para eu mudar para CWB, meu irmão mora ai e curto a cidade.

**Comment by matheus-rosin** [comment]
@kivervinicius massa, cara. Manda e-mail também, matheus@personaliza.com.br
